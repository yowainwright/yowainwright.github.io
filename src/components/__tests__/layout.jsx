import React from 'react';
import renderer from 'react-test-renderer';
import Template from '../layout';

describe('Template component', () => {
  it('renders correctly', () => {
    const node = <div />;
    const rendered = renderer.create(<Template children={node} />).toJSON() // eslint-disable-line

    expect(rendered).toMatchSnapshot();
  });
});
