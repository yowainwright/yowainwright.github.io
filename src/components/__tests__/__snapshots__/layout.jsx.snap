// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Template component renders correctly 1`] = `
<div
  className="gatsby-container"
>
  <nav
    className="social-nav"
  >
    <ul
      className="social-list"
    >
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://github.com/yowainwright"
        >
          Github
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://twitter.com/yowainwright"
        >
          Twitter
        </a>
      </li>
      <li
        className="social-list__item social-list__item--hidden"
      >
        <a
          className="social-list__link"
          href="https://www.npmjs.com/~yowainwright"
        >
          NPMJS
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://instagram.com/yowainwright"
        >
          Instragram
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="https://www.linkedin.com/in/jeffrywainwright/"
        >
          LinkedIn
        </a>
      </li>
      <li
        className="social-list__item social-list__item--showing"
      >
        <a
          className="social-list__link"
          href="/rss.xml"
        >
          RSS
        </a>
      </li>
    </ul>
  </nav>
  <nav
    className="site-nav"
    id="site-nav"
    itemType="http://schema.org/SiteNavigationElement"
    role="navigation"
  >
    <div
      className="site-nav__container"
    >
      <Link
        className="site-nav__main-item-link"
        to="/"
      >
        <div
          className="site-nav__main-item"
        >
          <h3
            className="site-nav__main-item-symbol"
          >
            j
          </h3>
          <h2
            className="site-nav__main-item-title"
          >
            Jeffry.in
          </h2>
        </div>
      </Link>
      <ol
        className="site-nav__items"
      >
        <li
          className="site-nav__item site-nav__item--about"
        >
          <Link
            className="site-nav__link site-nav__link--about"
            to="/about/"
          >
            About
          </Link>
        </li>
        <li
          className="site-nav__item site-nav__item--archive"
        >
          <Link
            className="site-nav__link site-nav__link--archive"
            to="/archive/"
          >
            Archive
          </Link>
        </li>
      </ol>
    </div>
  </nav>
  <div />
  <footer
    className="site-footer"
    itemType="http://schema.org/WPFooter"
    role="contentinfo"
  >
    <div
      className="site-footer__wrapper"
    >
      <div
        className="site-footer__col"
      >
        <figure
          className="figure--center site-footer__figure"
        >
          <img
            alt="Me smiling with a beard"
            className="media--circular"
            src="https://yowainwright.imgix.net/portraits/me-smiling.jpg?w=100&h=100&fit=crop&auto=format"
          />
        </figure>
        <p
          className="site-footer__content"
        >
          <Link
            className="site-footer__link"
            to="/"
          >
            jeffry.in 
            2021
          </Link>
        </p>
      </div>
    </div>
  </footer>
</div>
`;
